lib/amazing_print/json_helper.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+##
+# Helper module for hash_format: :json
+#
+module AmazingPrint
+  module JSONHelper
+    def json_awesome(object, is_key: false)
+      return inspector.awesome(object) unless options[:hash_format] == :json && object.respond_to?(:to_json)
+
+      if object.nil?
+        # Color null like we do nil
+        colorize(object.to_json, :nilclass)
+      elsif is_key && %w[BigDecimal Float Integer].include?(object.class.name)
```

Comment:
Why compare the `Class#name` instead of the `Class` objects themselves? Also, maybe these two arrays here should be put into constants in this module.

Reply:
Yeah let me try that

Reply:
Ah yeah the reason is that `BigDecimal` may not already be required and I don't think we should require objects just for this

Reply:
Updated